merge_spec.py

```diff
@@ -104,7 +104,9 @@ def transformation_of_openapi_v2(old_file_path, new_file_path):
         for method, endpoint in methods.items():
             # TODO: add types
             if "internal" not in endpoint["tags"] \
-                    and "admin" not in endpoint["tags"]:
+                    and "admin" not in endpoint["tags"] \
```

Comment:
Why did you use weird syntax to break lines? It looks so ugly.

Reply:
yep that looks bad, thats results of not having habits to use pep8 when writing and leaving it all to black ;)

Reply:
That's the result of ninja coding - once you say "ninja coding" & then "be like a scout - leave it cleaner than it was at the beginning" :)